src/Microsoft.Data.DataView/ColumnType.cs

```diff
@@ -0,0 +1,374 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable 420 // volatile with Interlocked.CompareExchange
+
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Threading;
+
+namespace Microsoft.ML.Data
```

Comment:
It's a bit strange to have an assembly named Microsoft.Data.* that contains the namespace Microsoft.ML.Data, no?  I'd have expected things to be in Microsoft.Data.*.

Reply:
The plan is to rename this namespace. But that is being done in a separate PR because it will touch almost every file in the repo.